demos/SDO-OH-EXF-dev-simple/README.md

```diff
@@ -0,0 +1,97 @@
+# Demonstration of SDO, OH, and EXF
+
+## Introduction
+
+This is a simple demonstration geared towards developers.  It begins with the Open Horizon all-in-one installation (including SDO services) in a single VM.  Running the modified SDO script, it will boot a simulated device and then install and configure an Open Horizon Agent.  The Agent will then contact the Management Hub and receive a proposed agreement to install and run an EdgeX Foundry (EXF) instance.  EXF, in turn, will be configured to run a Random Number Device service.
+
+## Requirements
+
+* An x86_64/amd64-based VM with Ubuntu 18.04 Server, 1 vCPU, 4GB RAM, and 20GB disk
```

Comment:
I have validated these steps on an HP MP9 G4 system with Intel Core i7 processor, 16GB memory, 500 GB storage, and Ubuntu 18.04.5 desktop.

Reply:
Good point.  I've added that info and left room for others to add to the list of systems that they've confirmed this on.